ugali/analysis/model.py

```diff
@@ -248,6 +248,7 @@ def __rrshift__(self, x):   return x >> self.__value__
     #def __ipow__(self, x):      self.set(self **x); return self
     # Casts
     def __nonzero__(self):      return self.__value__ != 0
+    def __bool__(self):      return self.__value__ != 0
```

Comment:
Better to have `__nonzero__` call `__bool__` rather than define them independently?